test/unit/sandbox-libraries/pm.test.js

```diff
@@ -940,4 +940,93 @@ describe('sandbox library - pm api', function () {
             }, function () {}); // eslint-disable-line no-empty-function
         });
     });
+
+    describe('execution', function () {
+        describe('.skipRequest~ ', function () {
```

Comment:
Add these as well:
1. it should not execute inside test Script
2. it should emit skip event when used in async context.

Reply:
Added